src/main/java/dk/kb/discover/api/v1/impl/DsDiscoverApiServiceImpl.java

```diff
@@ -167,12 +176,51 @@ public String solrSearch(String collection, String q, List<String> fq, Integer r
             SolrService solr = SolrManager.getSolrService(collection);
             // TODO: Pass the map of request parameters instead of all parameters as first class
             httpServletResponse.setContentType(solr.getResponseMIMEType(wt)); // Needed by SolrJ
+           
+             //Add filter query from license module.
+            DsLicenseApi licenseClient = getDsLicenseApiClient();
+            GetUserQueryInputDto licenseQueryDto = getLicenseQueryDto();
+            GetUsersFilterQueryOutputDto filterQuery = licenseClient.getUserLicenseQuery(licenseQueryDto);
+            log.info("filter query from licensemodule:"+filterQuery.getFilterQuery()); // in the start this is a very usefull log!            
```

Comment:
Please log the user attributes together with the resolved filter query for easier debugging.

Reply:
very useful indeed for the endless debugging to come,